### PR TITLE
layout idea list items

### PR DESCRIPTION
**Pull request by 2e2a**

### CODE REVIEW

File: apps/ideas/templates/meinberlin_ideas/includes/idea_list_tile.html [hunk 1]
```diff
@@ -1,7 +1,27 @@
...
+<div class="listItem">
+    <div class="listItem-stats">
+        <span class="rating-up is-read-only">
+            {{ idea.positive_rating_count }}
+            <i class="fa fa-chevron-up" aria-hidden="true"></i>
+        </span>
+        <span class="rating-down is-read-only">
+            {{ idea.negative_rating_count }}
+            <i class="fa fa-chevron-down" aria-hidden="true"></i>
+        </span>
```
Comment (xi) [inline apps/ideas/templates/meinberlin_ideas/includes/idea_list_tile.html]: I would prefer if these were wrapped in a `<div class="rating">`.

File: apps/ideas/templates/meinberlin_ideas/includes/idea_list_tile.html [hunk 2]
```diff
@@ -1,7 +1,27 @@
...
+        <span>
```
Comment (xi) [inline apps/ideas/templates/meinberlin_ideas/includes/idea_list_tile.html]: Something like `title="Comments"` would be good here.

File: meinberlin/assets/scss/_variables.scss [hunk 3]
```diff
@@ -11,10 +11,17 @@ $body-bg: #fff;
 $bg-secondary: #f6f6f6;
 $text-color: #222;
 $text-color-inverted: $body-bg;
+$text-color-introvert: lighten($text-color, 15%);
+$text-color-extrovert: darken($text-color, 10%);
```
Comment (xi) [inline meinberlin/assets/scss/_variables.scss]: I would prefer if we could do without these for now. We can add more colors once the designs are ready.

**Comment by xi** [comment]
LGTM. Feel free to merge once you considered my comments

**Comment by 2e2a** [comment]
applied comments ... merging.

**Pull request merged**